.github/ISSUE_TEMPLATE/package_issue.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Package Issue'
+description: Report issues and unexpected behaviour of packages with WinGet Client
+title: "[Package Issue]: Package.Identifer"
+labels: ["Issue-Bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new/choose)
+  - type: checkboxes
+    attributes:
+      label: Please confirm these before moving forward
+      description: Please confirm the following before posting your issue.
+      options:
+        - label: I have search for my issue and not found a work-in-progress/duplicate/resolved issue.
+          required: true
+        - label: I have not been informed if the issue is resolved in a preview version of the winget client.
+          required: true
+  - type: dropdown
+    attributes:
+      label: Category of the issue
+      description: What kind of issue are you running into?
+      options:
+        - Installation issue.
+        - Side-By-Side installation.
```

Comment:
These seem a bit interchangeable; As a user, more clarity would be helpful. What differentiates the two?

Reply:
I believe one would be error and the other would be unable to upgrade, it doesn't really matter but its better for the more experienced user